nodes/xdecor.lua

```diff
@@ -0,0 +1,112 @@
+
+-- Register wrench support for xdecor
+
+xdecor:register_repairable("wrench:wrench")
+
+local nodes = {
+	"xdecor:cabinet",
+	"xdecor:cabinet_half",
+	"xdecor:empty_shelf",
+	"xdecor:multishelf",
+}
+
+for _, nodename in pairs(nodes) do
+	wrench.register_node(nodename, {
+		lists = {"main"},
+	})
+end
+
+-- Cauldron
+
+nodes = {
+	"xdecor:cauldron_empty",
+	"xdecor:cauldron_idle",
+	"xdecor:cauldron_boiling",
+	"xdecor:cauldron_soup",
+}
+
+local function translated(player, text)
+	local player_name = player:get_player_name()
+	local lang_code = minetest.get_player_information(player_name).lang_code
+	return minetest.get_translated_string(lang_code, text)
+end
+
+for _, nodename in pairs(nodes) do
+	wrench.register_node(nodename, {
+		metas = {
+			infotext = wrench.META_TYPE_STRING,
+		},
+		description = function(pos, meta, node, player)
+			local str = translated(player, minetest.registered_nodes[node.name].infotext)
+			local i = string.find(str, "-")
+			return (i and i > 0) and string.sub(str, 1, i - 1) or str
+		end,
+
+	})
+end
```

Comment:
I don't think this is a good idea, not only will the picked up item be locked to the language of whoever picked it up, but the string manipulation might not work for some translations.

Maybe use the description of the empty cauldron?

Reply:
What's wrong with getting infotext normal way and leaving translations for xdecor itself?
Translation happens client side and normally you do not want to translate things unless it is strictly needed for server side processing, wrench does not do any server side processing.

Reply:
some infotexts seem to be too long for me. see
- https://github.com/minetest-mods/xdecor/blob/master/src/cooking.lua#L191
- https://github.com/minetest-mods/xdecor/blob/master/src/cooking.lua#L213

that's why I wanted to cut off the part after '-'.
but yes, it is cleaner to solve this in xdecor by a shorter description. 
i'll adjust this accordingly ...